laurent22/joplin
# Chore: Desktop: Fixes #9608: Fix comment style in Linux launcher file

# Summary

Fixes #9608 by following [the specification](https://specifications.freedesktop.org/desktop-entry-spec/latest/basic-format.html#comments)'s description of comments in `.desktop` files.

# Testing plan
1. Run the `Joplin_install_and_update.sh` script in GNOME to update Joplin.
2. Verify that Joplin can still be launched from its icon in the app list.
